kolibri/plugins/coach/assets/src/views/quizzes/CreateExamPage/sidePanels/QuizResourceSelection/index.vue

```diff
@@ -861,14 +861,17 @@
     }
   }
 
-  .alert-warning {
-    position: sticky;
+  .alert {
     top: 0;
-    z-index: 1;
     width: 100%;
     padding: 16px;
     margin-bottom: 16px;
     border-radius: 4px;
+  }
+
+  .alert.warning {
+    position: sticky;
+    z-index: 1;
```

Comment:
I know this was not introduced by this PR, but since we changed the cards to use KCard, cards thumbnails now have a z-index: 1, and this warning is appearing behind these thumbnails. Could you increase it to two now that we are modifying this part of the code? :sweat_smile:

![image](https://github.com/user-attachments/assets/b847f88c-90c3-4c0d-bbc3-30535f18b345)

Although I dont know if this is something we should fix in 0.18.0 cc: @rtibbles.

Reply:
Thanks for pointing this out! Since it wouldn't introduce any styling conflicts in this current PR, I went ahead and updated the z-index to 2.